### PR TITLE
[EDU-4564] refactor: refine Fingerprint guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4564]

### Changes

- Improve the Fingerprint guide, refining "embedding the script" instructions.

### Additional links

n/a.

[EDU-4564]: https://aziontech.atlassian.net/browse/EDU-4564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lucasfurno — APPROVED** [review]
Approved!

**Review by PabloDiehl — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx [hunk 1]
```diff
@@ -28,15 +28,30 @@ The browser script is a JavaScript code that runs on the web page you want to fi
...
+3.  Include the JavaScript file in your project folder.
+    1. In this example, the file name is `fingerprint-script.js`
+4. Then, add it to your HTML source, following this example:
+
+```bash
+<script src="https://example.com/fingerprint-script.js"></script>
+```
+
+Where: 
+- `https://example.com/` é o seu domínio.
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx:52]: Tradução: "é o seu domínio"?

File: src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx [hunk 2]
```diff
@@ -28,15 +28,30 @@ The browser script is a JavaScript code that runs on the web page you want to fi
...
+- `fingerprint-script.js` é o nome do seu arquivo JS.
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx:53]: Mesmo caso de tradução?

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx:52]:
```suggestion
- `https://example.com/` is your domain.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fingerprint.mdx:53]:
```suggestion
- `fingerprint-script.js` is your JavaScript file name.
```

**Pull request merged**